CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### NI-SCOPE
     * #### Added
+        * Exposed niScope_CalFetchDate, niScope_CalFetchTemperature, and niScope_CalFetchMiscInfo - [#1463}(https://github.com/ni/nimi-python/issues/1463)
```

Comment:
Talk about added functions in terms of their Python names. Remove word exposed. Remember this is read by Python users who should not need to know C APIs
When referring to an issue, don't paste full path, just write `#1463`.

Reply:
With the full path, I believe I was modeling past changelogs, but I've now fixed that.